### PR TITLE
added documentation keyword to top 25 languages

### PR DESCRIPTION
**Pull request by mahadwaseem123**
Hi @breck7 , documentation keyword is now there for top 25 languages.

### CODE REVIEW

File: database/things/csharp.pldb [hunk 1]
```diff
@@ -3,6 +3,7 @@ appeared 2000
 type pl
 creators Anders Hejlsberg
 website https://docs.microsoft.com/dotnet/csharp/language-reference/
+documentation https://learn.microsoft.com/en-us/dotnet/csharp/
```
Comment (breck7) [inline database/things/csharp.pldb:6]: nice!

File: database/things/css.pldb [hunk 2]
```diff
@@ -3,6 +3,7 @@ appeared 1996
 type stylesheetLanguage
 creators Håkon Wium Lie
 webRepl https://playcode.io/css/
+documentation https://devdocs.io/css/
```
Comment (breck7) [inline database/things/css.pldb:6]: wow devdocs is a pretty cool site. not sure if ive seen it before. nice find!

File: database/things/json.pldb [hunk 3]
```diff
@@ -3,6 +3,7 @@ appeared 2001
 type dataNotation
 creators Douglas Crockford
 website https://www.json.org/
+documentation https://www.json.org/json-en.html
```
Comment (breck7) [inline database/things/json.pldb:6]: we could probably drop the path

File: database/things/php.pldb [hunk 4]
```diff
@@ -3,6 +3,7 @@ appeared 1995
 type pl
 creators Rasmus Lerdorf
 website https://php.net
+documentation https://devdocs.io/php/
```
Comment (breck7) [inline database/things/php.pldb:6]: we should also add https://www.php.net/docs.php. `documentation` keyword should allow multiple, if we don't support that yet.

**Comment by breck7** [comment]
Amazing work @mahadwaseem123 !!

File: database/things/css.pldb (hunk 2 above)
Comment (mahadwaseem123) [reply database/things/css.pldb:6]: Yes @breck7 , it is a pretty cool site.

File: database/things/json.pldb (hunk 3 above)
Comment (mahadwaseem123) [reply database/things/json.pldb:6]: ok sure

File: database/things/php.pldb (hunk 4 above)
Comment (mahadwaseem123) [reply database/things/php.pldb:6]: Will definitely check that.

**Comment by mahadwaseem123** [comment]
Thank you @breck7 .

**Pull request merged**